user-org-service/app/utils/validator/uservalidator/UserModelValidator.java

```diff
@@ -0,0 +1,20 @@
+package utils.validator.uservalidator;
+
+import org.everit.json.schema.Schema;
+import org.json.JSONObject;
+import org.sunbird.util.LoggerEnum;
+import org.sunbird.util.ProjectLogger;
+import utils.validator.IModelValidator;
+import utils.validator.schema.SchemaFactory;
+
+public class UserModelValidator implements IModelValidator {
+
+  @Override
+  public void validate(JSONObject request, String uri) {
```

Comment:
validate method should throw except as well.

Reply:
sir it will throw and we are catching it at BaseController:handleRequest method

Reply:
But it's internally throwing it , that u know becuase u have written the code, by seeing code i can say this method is not throwing exception. think abt other person , who is going to use this code